src/main/java/com/wanderersoftherift/wotr/mixinextension/WotrGuiGraphics.java

```diff
@@ -0,0 +1,23 @@
+package com.wanderersoftherift.wotr.mixinextension;
+
+import net.minecraft.client.gui.Font;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.inventory.tooltip.TooltipComponent;
+import net.minecraft.world.item.ItemStack;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Optional;
+
+public interface WotrGuiGraphics {
+
+    void wotr$RenderTooltipLeft(
```

Comment:
I have a feeling that perhaps the before/after comparison displays shouldn't be tooltips, but this is fine.

Reply:
Yeah, same, just unsure how to show it otherwise atm.